examples/schema-registry/avro/src/main/java/com/hortonworks/registries/schemaregistry/examples/avro/KafkaAvroSerDesApp.java

```diff
@@ -47,17 +48,25 @@
 import java.util.Properties;
 import java.util.concurrent.TimeUnit;
 
+import static com.hortonworks.registries.schemaregistry.serdes.avro.AvroSnapshotSerializer.SERDES_PROTOCOL_VERSION;
+import static com.hortonworks.registries.schemaregistry.serdes.avro.SerDesProtocolHandlerRegistry.METADATA_ID_VERSION_PROTOCOL;
+
 /**
  * Below class can be used to send messages to a given topic in kafka-producer.props like below.
  *
  * KafkaAvroSerDesApp -sm -d yelp_review_json -s yelp_review.avsc -p kafka-producer.props
+ *
+ * If invalid messages need to be ignored while sending messages to a topic, you can set "ignoreInvalidMessages" to true
+ * in kafka producer properties file.
+ *
  */
 public class KafkaAvroSerDesApp {
     private static final Logger LOG = LoggerFactory.getLogger(KafkaAvroSerDesApp.class);
 
     public static final String MSGS_LIMIT = "msgsLimit";
     public static final String TOPIC = "topic";
     public static final int DEFAULT_MSGS_LIMIT = 50;
+    public static final String IGNORE_INVALID_MSGS = "ignoreInvalidMessages";
```

Comment:
nit: should probably be a dot separated property if its a property in a property file.